### PR TITLE
Better issue templates

### PR DESCRIPTION
**Pull request by acidicMercury8**
Changes proposed in this pull request:

- Added basic pull request template
- Rewrited issue templates to YAML

Related issue:

- #29

Agreements

- [x] I agree to follow this project's [__Contributing Guidelines__](https://github.com/hexalang/.github/blob/kawaii/CONTRIBUTING.md)
- [x] I agree to follow this project's [__Code of Conduct__](https://github.com/hexalang/.github/blob/kawaii/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Changes proposed in this pull request:
+
+<!--
+Please insert below the list of changes in this pull request. For example:
+
+- Change 1
```
Comment (PeyTy) [inline .github/PULL_REQUEST_TEMPLATE/pull_request.md:6]: Good, making examples as comments

File: .github/PULL_REQUEST_TEMPLATE/pull_request.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+-->
+
+-
+
+Related issue:
+
+<!--
+If this pull request is related to any issue, then put its number here please. For example:
+
+- #8
```
Comment (PeyTy) [inline .github/PULL_REQUEST_TEMPLATE/pull_request.md:18]: So this `#` is then converted to nice url automagically @acidicMercury8 ?

image

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+title: ''
+labels: #enhancement
+body:
+  - type: textarea
+    id: formulation
+    attributes:
+      label: Is your feature request related to a problem? Please describe
+      description: A clear and concise description of what the problem is
+      placeholder: |
+        I'm always frustrated when [...]
```
Comment (PeyTy) [inline .github/ISSUE_TEMPLATE/feature_request.yml]: Please change to positive (non-negative) vibes like `I had a situation, when [...]`

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
```
Comment (PeyTy) [inline .github/ISSUE_TEMPLATE/feature_request.yml:18]: Add extra `textarea` about how it affected the development. Like, what was the negative impact?

File: .github/PULL_REQUEST_TEMPLATE/pull_request.md (hunk 2 above)
Comment (acidicMercury8) [reply .github/PULL_REQUEST_TEMPLATE/pull_request.md:18]: @PeyTy yes, it was converted automatically

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 3 above)
Comment (acidicMercury8) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: @PeyTy done

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      required: true
+  - type: dropdown
+    id: device-type
+    attributes:
+      label: Device type
+      description: What type of device are you using?
+      multiple: false
+      options:
+        - label: Desktop
+        - label: Smartphone
```
Comment (PeyTy) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Is it even useful here? It's now like Hexa is a mobile app... and "Platform" has same info

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 4 above)
Comment (acidicMercury8) [reply .github/ISSUE_TEMPLATE/feature_request.yml:18]: @PeyTy done too

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 5 above)
Comment (acidicMercury8) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: @PeyTy okay, fixed already

**Review by PeyTy — APPROVED** [review]

**Pull request merged**